src/OidcClientSettings.ts

```diff
@@ -107,6 +107,11 @@ export interface OidcClientSettings {
      * Credentials used by fetch with the refresh request. (default: "same-origin")
      */
     refreshTokenCredentials?: "same-origin" | "include" | "omit";
+
+    /**
+     * Will check the content type header of the response of the revocation endpoint to match these passed values (default: ["application/json"])
+     */
+    revokeTokenContentTypes?: string[];
```

Comment:
I would like if it matches with JsonService constructor parameter `additionalContentTypes`. Alternative we could refactor that constructor parameter to `contentTypes`...

revokeTokenContentTypes -> revokeTokenAdditionalContentTypes
default: []